src/DefaultFormatter.php

```diff
@@ -80,7 +80,7 @@ public function format(\stdClass $tweet)
             '%in_reply_to_screen_name%' => $tweet->in_reply_to_screen_name,
             '%in_reply_to_status_id_str%' => $tweet->in_reply_to_status_id_str,
             '%in_reply_to_user_id_str%' => $tweet->in_reply_to_user_id_str,
-            '%place%' => $tweet->place,
+            '%place%' => $tweet->place->full_name,
```

Comment:
If `$tweet->place` can be `null`, shouldn't there be a check for that condition here?